docs/source/usersguide/settings.rst

```diff
@@ -0,0 +1,223 @@
+.. _usersguide_settings:
+
+==================
+Execution Settings
+==================
+
+.. currentmodule:: openmc
+
+Once you have created the materials and geometry for your simulation, the last
+step to have a complete model is to specify execution settings through the
+:class:`openmc.Settings` class. At a minimum, you need to specify a :ref:`source
+distribution <usersguide_source>` and :ref:`how many particles to run
+<usersguide_particles>`. Many other execution settings can be set using the
+:class:`openmc.Settings` object, but they are generally optional.
+
+.. _usersguide_run_modes:
+
+---------
+Run Modes
+---------
+
+The :attr:`Settings.run_mode` attribute controls what run mode is used when
+:ref:`scripts_openmc` is executed. There are five different run modes that can
+be specified:
+
+'eigenvalue'
+  Runs a :math:`k` eigenvalue simulation. See :ref:`methods_eigenvalue` for a
+  full description of eigenvalue calculations. In this mode, the
+  :attr:`Settings.source` specifies a starting source that is only used for the
+  first fission generation.
+
+'fixed source'
+  Runs a fixed-source calculation with a specified external source, specified in
+  the :attr:`Settings.source` attribute.
+
+'volume'
+  Runs a stochastic volume calculation.
+
+'plot'
+  Generates slice or voxel plots (see :ref:`usersguide_plots`).
+
+'particle_restart'
+  Simulate a single source particle using a particle restart file.
+
+
+So, for example, to specify that OpenMC should be run in fixed source mode, you
+would need to instantiate a :class:`openmc.Settings` object and assign the
+:attr:`Settings.run_mode` attribute::
+
+   settings = openmc.Settings()
+   settings.run_mode = 'fixed source'
+
```

Comment:
Is there a default run mode? If so, it should be mentioned here. If not, perhaps add the run mode as a required setting above, along with the source distribution and number of particles.